src/content/blog/2023/03/22/react-labs-what-we-have-been-working-on-march-2023.md

```diff
@@ -29,7 +29,7 @@ Since our last update, we have merged the [React Server Components RFC](https://
 
 The biggest change is that we introduced [`async` / `await`](https://github.com/reactjs/rfcs/pull/229) as the primary way to do data fetching from Server Components. We also plan to support data loading from the client by introducing a new Hook called `use` that unwraps Promises. Although we can't support `async / await` in arbitrary components in client-only apps, we plan to add support for it when you structure your client-only app similar to how RSC apps are structured.
 
-Now that we have data fetching pretty well sorted, we're exploring the other direction: sending data from the client to the server, so that you can execute database mutations and implement forms. We're doing this by letting you pass Server Action functions across the server/client boundary, which the client can then call, providing seamless RPC. Server Actions also give you progressively enhanced forms before JavaScript loads.
```

Comment:
Can we leave old blog posts unedited? I think it’s a bit confusing to change them.